docs/requirements.md

```diff
@@ -1,26 +1,26 @@
 
-1. Rely on open standards and be interoperable.
-1. Rely on tools and formats widely supported by web crawlers.
-1. Be easy to implement, with no economic or technical barriers.
+1. Rely on open standards  - especially Web standards - and be interoperable.
+1. Rely on tools and formats widely supported by publishers and TDM Actors.
+1. Minimize economic and technical barriers to implementation.
 
-As noted above, TDM and content indexing for search and discovery have different usage purposes. Therefore a prerequisite for any technical solutions that may be explored is that such solution shall enable rightsholders to reserve TDM rights without affecting access to other services such as content indexing and listing by search engines, nor affect content ranking in search pages results. 
+TDM and content indexing for search and discovery have different usage purposes. Therefore a prerequisite for any technical solutions that may be explored is that such solution shall enable rightsholders to reserve TDM Rights without affecting access to other services such as content indexing and listing by search engines, nor affect content ranking in search pages results. 
 
-## Primary requirements
+## Core requirements
 
-1. Specify how a rightsholder can declare TDM rights on a Web page he controls.
-1. Specify how a rightsholder can declare TDM rights on each individual Web resource he controls, when such Web resources are present in a Web page.
-1. Specify how the rightsholder can indicate whether a licence for TDM rights is available (in case TDM rights are reserved).
-1. Specify how a TDM actor can apply for a TDM license.
-1. Specify a machine-readable format for TDM licenses. 
+The solution shall:
 
-## Secondary requirements
+1. Specify how a rightsholder can declare the reservation of TDM Rights on each individual Web resource he controls.
+1. Specify how a rightsholder can indicate the location of human readable information indicating how to apply for a TDM License.
+1. Specify how a rightsholder can indicate the location of a TDM Licence associated with each individual Web resource he controls.
+1. Specify a machine-readable format for TDM Licenses. 
 
-Although this should not over-complexify the solution, it should still:
+## Additional Requirements for Advanced Solutions
 
-1. Allow the rightsholder to specify whether i) all TDM rights (other than for scientific research purposes) are reserved or ii) only TDM rights for commercial purposes are reserved  
-1. Specify how a rightsholder can indicate that TDM rights are reserved on random Web resources he controls (e.g. images and video files which may be found by crawlers outside of Web pages).
\ No newline at end of file
+Advanced user needs or sector specific practices may imply additional requirements. The additional features resulting from these additional requirements will not replace the code features resulting from core requirements, but will complement them. 
```

Comment:
Remove the word `code` (ie. "code features").

Reply:
Typo; we were meaning "core".